frontend/src/App.tsx

```diff
@@ -100,7 +100,6 @@ const App = (): React.ReactElement => {
                   path={Routes.EMAIL_ACTION}
                   component={EmailActionHandler}
                 />
-// TODO: Remove route for production
```

Comment:
Comment was giving me an error in rendering so I temp removed it

Reply:
pulling from master should resolve this!